src/3133R0_first_factor.md

```diff
@@ -196,6 +196,24 @@ There may be other overloads besides the above for `std::uint64_t`, as we'll dis
 However, in every case, the return type should be the same as the input type, because the result of
 the function can clearly never exceed its input.
 
+## Compiler iteration limits
+
+The compiler iteration limits that we mentioned above present an additional complication.  We have
+already seen that there are inputs that exceed these limits for naive algorithms, preventing
+compilation.  We don't yet know whether there are any inputs that would exceed the limits for the
+more efficient algorithms that we would like to use.  (After all, avoiding the need to research,
+tune, and implement these algorithms ourselves was a major reason for writing this paper!)
+
+Pessimistically, we'll assume that there do exist inputs that will exceed the compiler's preferred
+iteration limits.  If that happens, the compiler will need to circumvent these limits for
+`std::first_factor` in order to be compliant with the standard.
+
+We are open to suggestion as to the mechanism.  Perhaps the compiler can provide directives to
+disable these limits for _individual blocks of code_.  These directives could be either private (for
+compiler-internal use cases only), or public (for end users as well).  In any case, the most
+important point for this present proposal is that `std::first_factor` must be able to produce
+a result for every input at compile time.
```

Comment:
We do not have to suggest the mechanism. Each compiler can choose what they like. Also, suggesting that we might need to change the iteration limits in parts of the code might not be most lucky also.

I would rather write that the library function, at least when evaluated at compile time, may need some help from the compiler internals (e.g., this is [common for some type-traits](https://stackoverflow.com/questions/49901309/is-it-possible-to-have-a-library-implementation-of-is-trivially-constructible-wi)).